torch/jit/__init__.py

```diff
@@ -1153,6 +1165,18 @@ def __getattr__(self, attr):
                 return self.__getattr__('forward').graph
             return Module.__getattr__(self, attr)
 
+        def _is_attribute(self, attr, value):
```

Comment:
This seems brittle because it has to look through every other potential place where getattr looks to make sure it doesn't fall into one of these categories. Is there a better way to structure this?